backend/src/libs/packages/server-application/libs/helpers/error-info-label-mapper/error-info-label-mapper.ts

```diff
@@ -0,0 +1,7 @@
+const errorInfoLabelMapper: Record<string, string> = {
```

Comment:
```suggestion
const errorInfoLabelMap: Record<string, string> = {
```

Reply:
Bcos mapper would be suitable for function that has an input and return one value